retrieval/discovery/dns.go

```diff
@@ -32,6 +32,7 @@ const (
 	resolvConf = "/etc/resolv.conf"
 
 	dnsSourcePrefix = "dns"
+	DNSNameLabel    = clientmodel.MetaLabelPrefix + "dns_name"
```

Comment:
`dns_srv_name` to avoid ambiguity with the instance CNAMEs that the SRV record resolves to.